### DISCUSSION TITLE
JUNO探测器响应

### DISCUSSION THREAD

**Discussion by YiHui-Liu** [physics-data/faq#139] [Q&A] [answered]
>在实现探测器的响应的时候，最简单的方法是根据平方反比计算

这句话的意思是响应的幅度（反应能量）是根据光子从产生到激活PE经过的距离的平方反比来衰减吗？

**Comment by heroxbd** [comment]
@Berrysoft

**Comment by Berrysoft** [comment, marked as answer]
这不是衰减。考虑均匀立体角出射的光子，单位面积的通量一定与距离成平方反比。由于光路可逆，对于一个 PMT 而言，接收到不同位置的顶点传来的光子数，也一定与距离成平方反比。
